### PR TITLE
Update source-exchange-rates connector

### LINKED ISSUE #3541: Exchange API connector not working
## Expected Behavior
Connect to API and get data

## Current Behavior
image

## Logs
_If applicable, please upload the logs from the failing operation. For sync jobs, you can download the full logs from the UI by going to the sync attempt page and clicking the download logs button at the top right of the logs display window._

## Steps to Reproduce
1. 
1. 
1. 

## Severity of the bug for you
Very low / Low / **Medium** / High / Critical

## Airbyte Version
Found in the .env file in the root of the project.

## Connector Version (if applicable)
Found in the admin page in the UI in the respective Source or Destination tab.

## Additional context
_Environment, version, integration..._

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368060577387) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by Zirochkaa**
Since old API (https://api.ratesapi.io/) is gone we are using a new one - `http://api.exchangeratesapi.io/`.
Update code to support this new API.

## What
Closes #3541 issue.

## Pre-merge Checklist
* [x] _Run integration tests_
* [x] _Publish Docker images_

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200368060087111/1200375047657577) by [Unito](https://www.unito.io)

### CODE REVIEW

File: airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py [hunk 1]
```diff
@@ -38,26 +38,26 @@ class ExchangeRates(HttpStream):
     date_field_name = "date"
 
     # HttpStream related fields
-    url_base = "https://api.ratesapi.io/"
+    url_base = "http://api.exchangeratesapi.io/"
```
Comment (keu) [inline airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py:41]: I'm not sure why can't we move `v1` part here

**Review by keu — APPROVED** [review]
small comment, otherwise looks good to me

**Review by sherifnada — APPROVED** [review]
Can you double check it's being used in the source definitions yaml file? Also do a minor version bump for this one

File: airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py (hunk 1 above)
Comment (Zirochkaa) [reply airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py:41]: Sorry, I just changed the previous `api/` to `v1/` but you are right, will move `v1/` to `url_base`.

**Comment by Zirochkaa** [comment]
> Can you double check it's being used in the source definitions yaml file? Also do a minor version bump for this one

`source-exchange-rates` is not in the `source_definitions.yaml` file, only `source-exchangeratesapi-singer`, so I'll add it there.

File: airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py (hunk 1 above)
Comment (Zirochkaa) [reply airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py:41]: Done.

**Review by keu — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 2]
```diff
@@ -3,11 +3,12 @@
   dockerRepository: airbyte/source-asana
   dockerImageTag: 0.1.0
   documentationUrl: https://hub.docker.com/r/airbyte/source-asana
-- sourceDefinitionId: 9fed261d-d107-47fd-8c8b-323023db6e20
+  icon: asana.svg
```
Comment (jrhizor) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:6]: This shouldn't have been re-added here.

Comment (Zirochkaa) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:6]: Got it, sorry.

**Pull request merged**